custom/icds_reports/ucr/data_sources/dashboard/child_tasks.json

```diff
@@ -55,6 +55,19 @@
           "location_property": "_id"
         }
       },
+      {
+        "column_id": "supervisor_id",
+        "datatype": "string",
+        "type": "expression",
+        "is_nullable": true,
+        "expression": {
+          "location_id_expression": {
+            "type": "property_name",
+            "property_name": "owner_id"
+          },
+          "type": "location_parent_id"
```

Comment:
I think this will result in an extra db lookup to the locations table. `ancestor_location` (used for state_id) caches the entire hierarchy so its only one lookup per doc. That's not used in all UCRs, but should be in the newest ones

Reply:
updated to use `ancestor_location`